lib/file_opener.rb

```diff
@@ -0,0 +1,17 @@
+class FileOpener
+  def self.open_files(locations)
+    games = CSV.open locations[:games], headers: true, header_converters: :symbol
```

Comment:
I'm now thinking we can go a step further on this refactor. We have three CSV.open commands in a row, can we make it DRYer?